samples/downloadFolderWithTransferManager.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * @experimental
+ */
+
+// sample-metadata:
+//   title: Download Folder With Transfer Manager
+//   description: Downloads a folder in parallel utilizing transfer manager.
+//   usage: node downloadFolderWithTransferManager.js <BUCKET_NAME> <FOLDER_NAME>
+
+function main(bucketName = 'my-bucket', folderName = 'my-folder') {
+  // [START storage_download_folder_transfer_manager]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // The ID of your GCS bucket
+  // const bucketName = 'your-unique-bucket-name';
+
+  // The ID of the GCS folder to download
+  // const folderName = 'your-folder-name';
```

Comment:
does this support both relative and absolute? 
If it does can you add it to the comment